src/test/java/com/epam/eco/commons/jdbc/ConnectionTimeoutWrapperTest.java

```diff
@@ -0,0 +1,561 @@
+/*
+ * Copyright 2020 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.  You may obtain a copy
+ * of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.epam.eco.commons.jdbc;
+
+import java.sql.Array;
+import java.sql.Blob;
+import java.sql.CallableStatement;
+import java.sql.Clob;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.NClob;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLClientInfoException;
+import java.sql.SQLException;
+import java.sql.SQLWarning;
+import java.sql.SQLXML;
+import java.sql.Savepoint;
+import java.sql.Statement;
+import java.sql.Struct;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.Executor;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Uladzislau_Belykh
+ */
+public class ConnectionTimeoutWrapperTest {
+
+    @Test
+    public void testWrapTimeoutBelowZero() {
```

Comment:
Please try to use better naming convention to test methods. Name should explain what you're testing and what you're expecting. See https://qualitycoding.org/unit-test-naming/ 
I'd name this method testFailsOnNegativeTimeout().